simplemap.php

```diff
@@ -15,7 +15,7 @@
 #### CONSTANTS ####
 
 // Plugin Version Number
-define( 'SIMPLEMAP_VERSION', '2.4.11' );
+define( 'SIMPLEMAP_VERSION', '2.4.11.1' );
```

Comment:
__[code_sniffer]__
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->